src/translation/polish.c

```diff
@@ -772,7 +772,7 @@ static translation_string all_strings[] = {
     {TR_CONFIG_UI_SHOW_PARTIAL_GRID_AROUND_CONSTRUCTION, "Pokaż częściową siatkę mapy przy budowaniu" },
     {TR_OUT_OF_MONEY, "Brak pieniędzy" },
     {TR_CITY_MESSAGE_TITLE_EMPERORS_WRATH, "Gniew Cezara" },
-    {TR_CITY_MESSAGE_TEXT_EMPERORS_WRATH, "Wypadłeś z łask Cezara zatem zarządził Twoje aresztowanie. Jeżeli nie dasz rady przywrócić łaski Cezara, jego elitarni legioniści niedługo zaatakują Twoje miasto" },
+    {TR_CITY_MESSAGE_TEXT_EMPERORS_WRATH, "Wypadłeś z łask Cezara zatem zarządził Twoje aresztowanie. Jeżeli nie dasz rady odzyskać łask Cezara, jego elitarni legioniści niedługo zaatakują Twoje miasto" },
```

Comment:
chyba "Jeżeli nie dasz rady odzyska łask Cezara" by jednak lepiej brzmiało ...

Reply:
Powiedziałbym też że 'wypaść z łask' jest mało naturalnym określeniem. Po polsku chyba częściej stosuje się 'popaść w niełaskę'. Całe to zdanie jest jakieś dziwne!

```suggestion
    {TR_CITY_MESSAGE_TEXT_EMPERORS_WRATH, "Popadłeś w niełaskę u Cezara,i zarządził on Twoje aresztowanie. Jeśli nie zdołasz wrócić do łask Cezara, wkrótce jego elitarni legioniści zaatakują Twoje miasto." },
```